CMakeLists.txt

```diff
@@ -124,6 +126,42 @@ list(APPEND CMAKE_MODULE_PATH
   ${CMAKE_CURRENT_LIST_DIR}/third_party/abseil-cpp/absl/copts/
 )
 
+#-------------------------------------------------------------------------------
+# Cross compiling configuration
+#-------------------------------------------------------------------------------
+
+if(CMAKE_CROSSCOMPILING)
+  if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+    message(FATAL_ERROR "Cross compilation with Windows host unspported yet")
```

Comment:
```suggestion
    message(FATAL_ERROR "Cross compilation with Windows host unsupported yet")
```

nit: typo

Reply:
Revised. Hopefully this time it's better. :)